### PR TITLE
Patch bug: missing finally block in try-except structures

### PR DESCRIPTION
**Pull request by samuelchassot**
solves #98
I'll add tests

### CODE REVIEW

**Comment by Jarvx** [comment]
@samuelchassot  Thanks for the help. Would you mind submit this PR to this branch https://github.com/SMAT-Lab/Scalpel/tree/scalpel-CFG-for-final-clause.

It takes a much longer time to review changes for the main branch. For sure, this contribution will be merged into the next release.

**Comment by samuelchassot** [comment]
With pleasure :)
Yes, sure, I'll make the modification tomorrow when I'll add a test.

File: src/scalpel/cfg/builder.py [hunk 1]
```diff
@@ -430,6 +431,7 @@ def visit_Try(self, node):
         # Create a new block for the body of try.
         try_block = self.new_block()
         self.add_exit(self.current_block, try_block, ast.Constant(True))
+        print(node.orelse)
```
Comment (Jarvx) [inline src/scalpel/cfg/builder.py:434]: I think this is a dubugging satement

Comment (samuelchassot) [reply src/scalpel/cfg/builder.py:434]: Yes, definitely. I am adding a test and then will clean the code 👍 I'll mark the PR as ready.
I cannot run the tests locally so I will put it in `src`.
Also, I couldn't find any tests asserting the resulting CFG. Do you want me to write one for that case or do you prefer not testing this?

Comment (Jarvx) [reply src/scalpel/cfg/builder.py:434]: Please just do whatever you want :)  you can put tests right under the same folder 

I am also thinking to have some unit tests for all control flow statements next (might just be using all the example code in user guide). Currently, the CFG is tested by some other modules.

Comment (samuelchassot) [reply src/scalpel/cfg/builder.py]: sounds good! :)

**Comment by samuelchassot** [comment]
Let me know if any changes are needed :)

**Review by Jarvx — APPROVED** [review]

**Comment by Jarvx** [comment]
Thanks for fixing the problem and adding so many test cases. I just realized the code I've written down before is very chaotic. I will refactor the CFG again and make it into the release branch soon. Please feel free to submit any issue reports.

**Comment by samuelchassot** [comment]
You're welcome! I see, that's great!

**Pull request merged**